blog/Migration-to-Fable2.md

```diff
@@ -0,0 +1,160 @@
+            resolve("./node_modules/")
+        ]
+    },
+    ```
+- Remove any reference to `fable-utils` like:
+    - `const fableUtils = require("fable-utils");`
+    - `fableUtils.resolveBabelOptions`
+- Convert your `babelOptions` and any thing related to `babel-` to use `@babel` scope if available
+
+    **babel options**
+    ```js
+    var babelOptions = fableUtils.resolveBabelOptions({
+        presets: [
+            ["env", {
+                "targets": {
+                    "browsers": ["last 2 versions"]
+                },
+                "modules": false
+            }]
+        ]
+    });
+    ```
+    becomes
+    ```js
+    var babelOptions = {
+        presets: [
+            ["@babel/preset-env", {
+                "targets": {
+                    "browsers": ["last 2 versions"]
+                },
+                "modules": false
+            }]
+        ]
+    };
+    ```
+
+    **babel polyfill**
+
+    "babel-polyfill", becomes "@babel/polyfill"
+
+- Because Fable 2, now create a `.fable` folder you are not forced anymore to use **absolute** filepath. All the filepath are now relative to the nearest `package.json` found by Fable. When running Fable, it will tell you what it's *current working directory* is.
+    For example:
+
+        Fable (2.0.1) daemon started on port 56569
+        CWD: /Users/maximemangel/Workspaces/Github/MangelMaxime/Fulma/templates/Content
+
+You can find [here a webpack.config.js](https://github.com/fable-compiler/webpack-config-template/blob/master/webpack.config.js) file to serve as a template for yours.
+
+If needed you can also take a look at the Fulma.Minimal diff [here](https://github.com/MangelMaxime/Fulma/commit/24c9269fc08eac9c66e774aec4a5fe4132968805#diff-b81ec06774f7dfd70d0cac66cf146951).
+
+## Update fable nuget package
+
+Run `.paket/paket.exe update` to install the latest version of your dependencies. You check that you are using the correct version of the package by comparing with this list:
+
+*At the time of writing is are the version retrieved*
+
+**Common to any Fable porject**
+
+<div style="max-width: 300px">
+
+| Package | Version |
+|---|---|
+| Fable.Core | 2.0.0 |
+| dotnet-fable | 2.0.1 |
+
+</div>
+<br/>
+
+**For projects using Elmish**
+
+<div style="max-width: 300px">
+
+| Package | Version |
+|---|---|
+| Fable.React | 4.0.1 |
+| Fable.Elmish | 2.0.0 |
+| Fable.Elmish.React | 2.0.0 |
+| Fable.Elmish.Browser | 2.0.0 |
+| Fable.Elmish.Debugger | 2.0.0 |
+| Fable.Elmish.HMR | 2.0.0 |
+
+</div>
+<br/>
+
+**General libraries** *you can send a PR if you think some commonly used library are missing*
+
+<div style="max-width: 300px">
+
+| Package | Version |
+|---|---|
+| Fable.PowerPack | 2.0.1 |
+| Fulma | 1.0.0 |
+| Fulma.Extensions | 1.0.0 |
+| Thoth.Json | 2.0.0 |
+
+</div>
+
+## Breaking changes
+
+Fable 2 introduce severals breaking changes, it will inform you by displaying an *obsolete* warning or by generating a warning in the ouput.
```

Comment:
Most important changes are:

- `PojoAttribute` is not needed anymore.
- `PassGenericsAttribute` is also deprecated. If you need to resolve a generic argument, please inline the function. If you only need the `Type` of a generic argument, you can use [the `Fable.Core.Inject` attribute with `ITypeResolver`](https://github.com/fable-compiler/Fable/blob/d0f09bb74524c03d200249ea12906e426e170b44/tests/Main/ReflectionTests.fs#L402-L405).
- **JSON serialization**: ...